src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -175,110 +263,145 @@ private WordEmbeddingsTransform(IHost host, ModelLoadContext ctx, IDataView inpu
             }
 
             Host.CheckNonWhiteSpace(_modelFileNameWithPath, nameof(_modelFileNameWithPath));
-            _currentVocab = GetVocabularyDictionary();
-            _outputType = new VectorType(NumberType.R4, 3 * _currentVocab.Dimension);
-            Metadata.Seal();
         }
 
-        public static WordEmbeddingsTransform Create(IHostEnvironment env, ModelLoadContext ctx, IDataView input)
+        public static WordEmbeddingsTransform Create(IHostEnvironment env, ModelLoadContext ctx)
         {
             Contracts.CheckValue(env, nameof(env));
             IHost h = env.Register(RegistrationName);
             h.CheckValue(ctx, nameof(ctx));
```

Comment:
>h.CheckValue(ctx, nameof(ctx)); [](start = 12, length = 31)

I think we have tendency to validate version info for model as well
ctx.CheckAtModel(GetVersionInfo());

Dunno why it's not present here.

Reply:
Yes, model checking is not there before I started converting into estimator. I will add.

---
In reply to: [218251360](https://github.com/dotnet/machinelearning/pull/928#discussion_r218251360) [](ancestors = 218251360)